packages/react-components/src/components/MessageStatusIndicator.test.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { MessageStatusIndicator } from './MessageStatusIndicator';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { mountWithLocalization, createTestLocale } from './utils/testUtils';
+import { TooltipHost } from '@fluentui/react';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('MessageStatusIndicator should work with localization', () => {
+  test('Should localize tooltip text', async () => {
+    const messageStatusIndicatorStrings = {
+      seenTooltipText: Math.random().toString(),

```

Comment:
Soft recommendation: I'd recommend against using random strings. Unittests should be as deterministic as possible. The randomness here _should be_ harmless, but still.

Fuzz testing is a useful tool. But they have different use than unit-tests. If we want fuzz testing, we should set it up to run continuously outside of CI. If the fuzz test finds a bug, we should roughly convert the _input that failed_ into a deterministic unit-test so that it _always fails_, then fix that bug to make the unittest pass.

Reply:
My intention was to test that ANY string defined in `XString` can be localized and so much to provide invalid, unexpected, or random data as fuzz testing is defined to be. But I don't mind changing the randomness of the unit tests. DO you prefer hardcoded strings to test with?

Reply:
It's very easy to ignore unittest failure in CI if it fails only some of the time. So yes, deterministic unittests are preferred.